tests/models/unit.py

```diff
@@ -49,11 +49,11 @@ def _update_translation(store, item, new_values, sync=True):
 
 @pytest.mark.django_db
 def test_getorig(af_tutorial_po):
-    """Tests that the in-DB Store and on-disk Store match by checking that
+    """Tests that the inDB Store and ondisk Store match by checking that
```

Comment:
If there's something wrong with using an hyphen in compound constructions, I'd be interested to know why.